04-Multi-Band-Raster.Rmd

```diff
@@ -117,104 +126,92 @@ par(original_par) # go back to original par
 
 ```
 
-#Other Types of Multi-band Raster data
+#Other Types of Multi-band Raster Data
 
 Multi-band raster data might also contain:
```

Comment:
yes - here i'm still calling them bands. let's just be consistent with the language. thank you for catching this!